input_schema.json

```
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "array",
+    "items": {
+      "type": "object",
+      "properties": {
+      "groupId": {
+          
+          "description": "Group identifier",
+          "pattern": "^[a-zA-Z0-9_-]+$"
+      },
+      "sampleId": {
+          "description": "Sample identifier that includes the group ID",
+          "pattern": "^([a-zA-Z0-9_-]+)$"
+      },
+      
+      "fastqbase": { "type": "string" },
+      "ref": { "type": "string" },
+      "parentId": { "type": "string" }
+
+      }
+    }
```

Not sure if my syntax is correct, but I assume it's good to fail here if not all columns are provided?
```suggestion
      },
      "required": ["groupId", "sampleId", "fastqbase", "ref", "parentId"]
    }
```